testserver/minecraft.js

```diff
@@ -0,0 +1,68 @@
+// Require Node.js standard library function to spawn a child process
+var spawn = require('child_process').spawn;
+var minecraftServerProcess;
```

Comment:
This is going to be a changing variable, so you should use `let` here.

Reply:
Done